### PR TITLE
WIP #495 async migration

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
to be removed and made in fork

### CODE REVIEW

File: Evaluator/RealTime/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#  Drakkar-Software OctoBot
```
Comment (Herklos) [inline Evaluator/RealTime/__init__.py]: Why adding license in this PR ?

Comment (GuillaumeDSM) [reply Evaluator/RealTime/__init__.py]: wrong target branch, fixed now

File: tests/Evaluator/Strategies/test_full_mixed_strategies_evaluator.py [hunk 2]
```diff
@@ -1,43 +1,63 @@
...
-    def test_slow_downtrend(strategy_tester):
-        strategy_tester.run_test_slow_downtrend(-9.5, -10.5, -20.5, 0)
+    async def test_slow_downtrend(strategy_tester):
+        await strategy_tester.run_test_slow_downtrend(-9.7, -9, -21.5, 0)
 
     @staticmethod
-    def test_sharp_downtrend(strategy_tester):
-        strategy_tester.run_test_sharp_downtrend(-14, -21.5)
+    async def test_sharp_downtrend(strategy_tester):
+        await strategy_tester.run_test_sharp_downtrend(-14.5, -21.5)
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply tests/Evaluator/Strategies/test_full_mixed_strategies_evaluator.py]: Evaluation update is now after all timeframe update at any given time instead of 1tf update => new orders, next tf update (in the same time) => new order or not.
Basically, now order update is after all required tf are updated (if tf to update are 30min AND 1h, order update is after both are updated instead of 30min => order update, 1h => order update.
This is why there are very small changes sometimes in results

**Comment by Herklos** [comment]
To be merged after OctoBot/feature-async merged into dev.

File: tests/Evaluator/Strategies/test_full_mixed_strategies_evaluator.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Herklos) [reply tests/Evaluator/Strategies/test_full_mixed_strategies_evaluator.py]: OK thanks for your explanation.

File: Evaluator/RealTime/__init__.py (hunk 1 above)
Comment (Herklos) [reply Evaluator/RealTime/__init__.py]: Perfect Thank you 👌

File: Evaluator/RealTime/instant_fluctuations_evaluator.py [hunk 3]
```diff
@@ -2,7 +2,6 @@
 OctoBot Tentacle
 
 $tentacle_description: {
-    "package_name": "OctoBot-Tentacles",
```
Comment (Herklos) [inline Evaluator/RealTime/instant_fluctuations_evaluator.py]: Why did you remove this line ?

Comment (GuillaumeDSM) [reply Evaluator/RealTime/instant_fluctuations_evaluator.py]: Because if was a mistake to add it in the first place, it's an auto generated field by the tentacle manager
